beacon_node/eth2_libp2p/src/rpc/rate_limiter.rs

```diff
@@ -185,7 +185,7 @@ impl RPCRateLimiter {
     pub fn allows<T: EthSpec>(
         &mut self,
         peer_id: &PeerId,
-        request: &RPCRequest<T>,
+        request: &RpcRequestContainer<T>,
```

Comment:
Does the rate limiter really need to know about the container?. 
We send a request to the behaviour, the behaviour sends it to the handler, the handler upgrades it to a container and creates streams, once it gets a response we return the response. 
IIRC the rate limiter occurs at the behaviour level.

Reply:
Fixed.